CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 1.16.0 (September 3, 2024)
+
+ * [#4289](https://github.com/redhat-developer/vscode-openshift-tools/issues/4289) Fixed slow loading of the Applicatoin Explorer
+ * [#4357](https://github.com/redhat-developer/vscode-openshift-tools/pull/4357) Allow commands to work with .yml extension
```

Comment:
I would make it `.yml` (with the back ticks)

Reply:
ACK